scripts/e2e-test.sh

```diff
@@ -4,8 +4,11 @@ set -eux
 
 SCRIPTDIR=$(dirname "$(realpath "$0")")
 # new tests should be added before the replica_pod_remove test
-#TESTS="install basic_volume_io csi replica rebuild node_disconnect/replica_pod_remove uninstall"
-TESTS="install basic_volume_io csi uninstall"
+#DEFAULT_TESTS="install basic_volume_io csi replica rebuild node_disconnect/replica_pod_remove uninstall"
+DEFAULT_TESTS="install basic_volume_io csi uninstall"
```

Comment:
I would rename the var to something like basic and extended tests. It sorta feels wrong, but I'm not a native speaker.
```suggestion
# DEFAULT_TESTS are run for each PR, LONG_TESTS only daily (see --long option, check Jenkinsfile for current schedule)
DEFAULT_TESTS="install basic_volume_io csi uninstall"
```

Reply:
I would not make any assumption about how often the tests run or mention anything about PR testing here. That's a matter of jenkins configuration. This script provides a way to run e2e tests regardless if they run in Jenkins, on local computer or somewhere else. As for the name of the variable itself, if you agree, since this concern is more about how to name variables, I will defer to @blaisedias 's opinion if things are understandable enough as they are named now. That can be certainly changed and will do that if there is agreement on what should be the new names. @blaisedias  any opinion on this?

Reply:
I kind of prefer extended to long, but both names convey something useful about the set of tests being run.
If long... is used then it is natural to expect that short... is defined.
Whereas extension implies that normal set + some more, which is the case.
It just so happens that the extended set now has long running tests. 
There is a case to rename the directory test/e2e/nightly to test/e2e/long or test/e2e/extended for consistency.

Reply:
I think we arrived to conclusion then. Let's use "extended" word then. I will update the PR 👍

Reply:
done